go/vt/vtgate/vtgate.go

```diff
@@ -142,6 +146,7 @@ func registerFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&enableSchemaChangeSignal, "schema_change_signal", enableSchemaChangeSignal, "Enable the schema tracker; requires queryserver-config-schema-change-signal to be enabled on the underlying vttablets for this to work")
 	fs.StringVar(&schemaChangeUser, "schema_change_signal_user", schemaChangeUser, "User to be used to send down query to vttablet to retrieve schema changes")
 	fs.IntVar(&queryTimeout, "query-timeout", queryTimeout, "Sets the default query timeout (in ms). Can be overridden by session variable (query_timeout) or comment directive (QUERY_TIMEOUT_MS)")
+	fs.StringSliceVar(&globalKeyspaceNames, "global-keyspace", globalKeyspaceNames, "Global routing behavior applies when no default keyspace is set on a connection, or when a global keyspace is used. Multiple global keyspaces may defined by specifying this flag multiple times.")
 	fs.StringVar(&queryLogToFile, "log_queries_to_file", queryLogToFile, "Enable query logging to the specified file")
```

Comment:
why do we need multiple names, I think 1 should be enough to switch to global routing.

Reply:
We don't, not sure why I did this. I think we could also hardcode a global ks name if we can be sure that it won't conflict with existing user db names. Do you think `@global` could work?